packages/http/src/mocker/negotiator/__tests__/NegotiatorHelpers.spec.ts

```diff
@@ -84,6 +95,24 @@ describe('NegotiatorHelpers', () => {
       });
 
       describe('and has no static contents', () => {
+        it('returns an empty payload response', () => {
```

Comment:
Let's put a word or two about the fact we're looking for a response for an invalid Request